src/components/organisms/ChatSidebar/ChatSidebar.scss

```diff
@@ -12,13 +12,13 @@ $header-height: 45px;
   right: 0;
   width: $chat-sidebar-width;
   height: calc(100vh - 96px);
-  background-color: $saturated-black;
+  @include background-color($saturated-black);
   z-index: z(sidebar);
 
   &--expanded {
     transform: translateX(0);
     transition: transform 0.3s;
-    background-color: rgba($saturated-black, 0.9);
+    @include background-color(rgba($saturated-black, 0.9));
```

Comment:
NOTE: function sets the needed CSS --variables for the descendant indicator border to preclude the direct parent's paint.